cmd/ltctl/loadtest.go

```diff
@@ -156,3 +156,49 @@ func RunLoadTestStatusCmdF(cmd *cobra.Command, args []string) error {
 
 	return nil
 }
+
+func RunInjectActionCmdF(cmd *cobra.Command, args []string) error {
+	config, err := getConfig(cmd)
+	if err != nil {
+		return err
+	}
+
+	t, err := terraform.New("", config)
+	if err != nil {
+		return fmt.Errorf("failed to create terraform engine: %w", err)
+	}
+
+	action := args[0]
+
+	status, errInject := t.InjectAction(action)
+	if errInject != nil {
+		fmt.Println("Failed to inject action ", action, ": ", errInject)
+		return errInject
+	}
+
+	tfOutput, err := t.Output()
+	if err != nil {
+		return err
+	}
+
+	prometheusURL := fmt.Sprintf("http://%s:9090", tfOutput.MetricsServer.PublicIP)
+	helper, err := prometheus.NewHelper(prometheusURL)
+	if err != nil {
+		return fmt.Errorf("failed to create prometheus.Helper: %w", err)
+	}
+
+	errInfo, err := getErrorsInfo(helper, status.StartTime)
+	if err != nil {
+		return err
+	}
+
+	usersCount, err := getUsersCount(helper)
+	if err != nil {
+		return err
+	}
+
+	printCoordinatorStatus(status, errInfo, usersCount)
```

Comment:
I guess it's not necessary.  I was just being consistent with the other commands.  I think I'll remove it, as it may be misleading since the results of the action are likely not reflected in these numbers at this point.